Issue/9/no need for bashrc

arl opened the pull request:
Remove the need to modify `$PROMPT_COMMAND` in user `.bashrc`
`$PROMPT_COMMAND` is now modified in tmux-gitbar.conf
As we still need an absolute path, we require the user to set it **if and only if**
the user doesn't install tmux-gitbar to the default location `~/.tmux-gitbar`

fixes #9

arl committed:
Rename tmux-gitbar.sh into update-gitbar

arl committed:
Chmod +x update-gitbar

arl committed:
Convert update-gitbar into callable script

arl committed:
Add tmux-gitbar.tmux

- tmux-gitbar.tmux is meant to be source'd from tmux.conf. Its sole
 purpose, is to modify PROMPT_COMMAND so that update-gitbar is called
 (with absolute path)

- In case tmux-gitbar is not installed in the default location,
 ~/.tmux-gitbar, it is required to define TMUX_GITBAR_DIR in tmux.conf,
 before sourcing tmux-gitbar.tmux

arl committed:
Add vim modelines

arl committed:
Create and call load_config function

aurelien-rainone commented:
Before merging, still need to :

 - rewrite README to indicate that there is no need for .bashrc
 - add a changelog section,
 - write somehere that its a breaking change (for people having a previous version and that added
 anything to their `.bashrc`
 - new tag: v0.2 or v1.0??

arl committed:
Update README

 - changelog
 - breaking change warning
 - rewrite installing section with new instructions
